cxotime/cxotime.py

```diff
@@ -9,7 +9,11 @@
 from astropy.time import Time, TimeCxcSec, TimeYearDayTime, TimeDecimalYear
 from astropy.time.utils import day_frac
 from astropy.utils import iers
-from astropy import _erfa as erfa
+# in astropy versions < 4.2, erfa was an astropy private package:
+try:
+    import erfa
+except:
```

Comment:
Flake8 check fails with "E722 do not use bare 'except'"

I'm not sure what we've decide on for flake8 and this bare exception.  I've shut up flake8 by doing "except Exception" but it seems that has no more value than doing a " # noqa " on it.  Should we just add E722 to our ignored flake8 checks?

Reply:
argh, right. This rule is one that bugs me. I just edited the except statement to catch the specific exception.

Reply:
with this clause, if the import itself fails, then it does not fall back. That might be the right behavior anyway.

Reply:
I've got no strong opinion.  If you are happy with ModuleNotFoundError for this case that's fine.

Reply:
I have no strong opinion either! I personally would have preferred # noqa

I'm sure @taldcroft has a preference.